### ISSUE TITLE
tfe_workspace resource does not support auto-destroy features

### ISSUE THREAD

**Issue by jakebracey** [hashicorp/terraform-provider-tfe#1157]
#### Terraform version

```plaintext
v1.5.7
```

#### Terraform Configuration Files

```terraform
resource "tfe_workspace" "test" {
  name         = "my-workspace-name"
  organization = "my-organization-name"
}
```

#### Debug Output

```plaintext
N/A
```

#### Expected Behavior

We manage terraform cloud workspaces completely with this provider. We would like to manage the auto-destroy features. 
#### Actual Behavior

When managing the tfe_workspace resource, we are unable to set auto-destroy setting to enable destroy at a specific time or if inactive after a set period of time.

**Comment by thrashr888** [comment]
Hi! We're planning on adding this soon. It might look something like this:

```hcl
resource "tfe_organization" "test-organization" {
  name  = "my-org-name"
  email = "admin@company.com"
}

resource "tfe_workspace" "test" {
  name                           = "my-workspace-name"
  organization                   = tfe_organization.test-organization.name
  auto-destroy-activity-duration = "18h"
}
```

And it might be easier to set it at the project level instead, like so:

```hcl
resource "tfe_project" "test" {
  organization = tfe_organization.test-organization.name
  name = "projectname"
  auto-destroy-activity-duration = "18h"
}
```

Would that work for you?

**Comment by jakebracey** [comment]
That would be great. I manage our organization at the workspace level so
not as worried about the project setting.

Jake

**Comment by thrashr888** [comment]
This was also shipped last week. Duration setting will come next. https://github.com/hashicorp/terraform-provider-tfe/pull/1354

**Comment by notchairmk** [comment]
Closing since all workspace auto-destroy features should be included in [v0.57.0](https://github.com/hashicorp/terraform-provider-tfe/releases/tag/v0.57.0)
